semgrep-core/src/core/Flag_semgrep.ml

```diff
@@ -23,8 +23,6 @@ let filter_irrelevant_rules = ref false
 (* check for identifiers before attempting to match a stmt or stmt list *)
 let use_bloom_filter = ref true
```

Comment:
so this should be called use_set_filter?

Reply:
Or you prefer to consistently keep the bloom_filter legacy name?

Reply:
Yeah, I'm used to the legacy name :)